File: Estrutura_Repetição/C#/Ex1074/Main.cs
using System;

class MainClass {
    public static void Main(string[] args) {
        int valores = Convert.ToInt32(Console.ReadLine());

        for (int i = 0; i < valores; i++) {
            int v = Convert.ToInt32(Console.ReadLine());
            if (v != 0) {
                if (v % 2 == 0){
                    if (v > 0) {
                        Console.WriteLine("EVEN POSITIVE");
                    } else {
                        Console.WriteLine("EVEN NEGATIVE");
                    }
                } else {
                    if (v > 0) {
                        Console.WriteLine("ODD POSITIVE");
                    } else {
                        Console.WriteLine("ODD NEGATIVE");
                    }
                }
            } else {
                Console.WriteLine("NULL");
            }
        }
    }
}

File: Estrutura_Repetição/C/Ex1074/Main.c
#include <stdio.h>

int main() {
    int v = 0, valores;
    scanf("%d", &valores);

    for (int i = 0; i < valores; i++) {
        scanf("%d", &v);
        if (v != 0) {
            if (v % 2 == 0) {
                if (v > 0) {
                    printf("EVEN POSITIVE\n");
                } else {
                    printf("EVEN NEGATIVE\n");
                }
            } else {
                if (v > 0) {
                    printf("ODD POSITIVE\n");
                } else {
                    printf("ODD NEGATIVE\n");
                }
            }
        } else {
            printf("NULL\n");
        }
    }

    return 0;
}